libs/audit-service/build.gradle

```diff
@@ -0,0 +1,55 @@
+plugins {
+	// Apply the java-library plugin for API and implementation separation.
+	id 'java-library'
+	id "idea"
+	id "jacoco"
+}
+
+repositories {
+	mavenCentral()
+}
+
+def var = dependencies {
```

Comment:
What does declaring `var` give you?

Reply:
trying to remember....added before to resolve issue with gradle build. Removed now.